dev/attest/package.json

```diff
@@ -21,7 +21,8 @@
         "!*.tsbuildinfo"
     ],
     "dependencies": {
-        "ts-morph": "18.0.0"
+        "ts-morph": "18.0.0",
+        "@arktype/utils": "workspace:^0.0.0"
```

Comment:
I'm used to using `workspace:*`.

Is there a reason to use this instead?

Reply:
Ah, nope -- we should use `workspace:*` so we don't have to update these every time we publish. Good catch